common/src/main/java/xyz/wagyourtail/jsmacros/client/mixins/access/MixinBossBarHud.java

```diff
@@ -15,7 +15,7 @@ public class MixinBossBarHud implements IBossBarHud {
     
     @Shadow
     @Final
-    private Map<UUID, ClientBossBar> bossBars;
+    Map<UUID, ClientBossBar> bossBars;
```

Comment:
these used to be private in a previous mc version, and it doesn't break by being the wrong access...
yeah, I guess it should be fixed, but its annoying for backporting

Reply:
I will change it back, but I would still recommend doing a small refactor to update the code to the latest Java version, i.e. instanceof pattern matching, better switch statements, toList(), Mixin visibilities, ... and then just reverting the commit for any older Java version.